packages/11ty/_plugins/transforms/outputs/epub/transform.js

```diff
@@ -41,6 +41,29 @@ module.exports = function(eleventyConfig, collections, content) {
     })
   }
 
+  /**
+   * Removes preceding slashes from asset paths
+   * @param {HTMLElement} element 
+   */
+  const transformPaths = (element) => {
+    const images = element.querySelectorAll('img')
+    const links = element.querySelectorAll('a')
+    const heros = element.querySelectorAll('.quire-page__header.hero, .quire-cover__overlay')
+    heros.forEach((item) => {
+      item.style.backgroundImage = item.style.backgroundImage.replace(/(?<=url\()\//, '')
+    })
+    images.forEach((item) => {
+      const src = item.getAttribute('src')
+      if (!src) return
+      item.setAttribute('src', src.replace(/^\//, ''))
+    })
+    links.forEach((item) => {
+      const href = item.getAttribute('href')
+      if (!href) return
+      item.setAttribute('href', href.replace(/^\//, ''))
+    })
```

Comment:
wondering if we should also handle `<video>` tags here, as they are supported in EPUB3?